charts/pega/README.md

```diff
@@ -640,8 +640,7 @@ dds:
 
 ## Search deployment
 
-Use the `pegasearch` section to configure a deployment of ElasticSearch for searching Rules and Work within Pega.  This deployment is used exclusively for Pega search, and is not the same ElasticSearch deployment used by the EFK stack or any other dedicated service such as Pega BI.
-
+Use the `pegasearch` section to configure the source ElasticSearch service that the Pega Platform deployment uses for searching Rules and Work within Pega. The ElasticSearch service defined here is not related to the ElasticSearch deployment if you defined an EFK stack for logging and monitoring in your Pega Platform deployment.
```

Comment:
one small tweak>
if you **_also define_** an EFK stack for logging and monitoring in your Pega Platform deployment.

Reply:
done.